addons/xeh/fnc_preInit.sqf

```diff
@@ -25,6 +25,18 @@ SLX_XEH_DisableLogging = uiNamespace getVariable ["SLX_XEH_DisableLogging", fals
 
 XEH_LOG("PreInit started. v" + getText (configFile >> "CfgPatches" >> "cba_common" >> "versionStr"));
 
+#ifdef RECOMPILE
+if (isNil "CBA_fnc_compileFunction") then {
+    INFO("Recompiling all CBA Functions [for use in threeden]");
+    { 
+        { 
+            private _fncName = format ["CBA_fnc_%1", configName _x]; 
+            _fncName call BIS_fnc_recompile; 
+        } forEach (configProperties [_x]); 
```

Comment:
Before you change this, can this config or the super config contain any other tokens besides classes? Looks like it according to https://community.bistudio.com/wiki/Arma_3_Functions_Library. Though we likely not make use of them.